### PR TITLE
Attempt to simulate WordPress' HTML transformation

### PR DESCRIPTION
**Pull request by andybalaam**
by inserting a `<p/>` between each block of text at the top level of a post.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -96,6 +99,8 @@ fn convert(input_file: PathBuf, output_dir: PathBuf, fs: &impl Fs) -> Result<()>
 
                 let markdown = parse_html(item.content());
                 debug!("{}", markdown);
+                let html = transform_html(item.content());
```
Comment (TatriX) [inline src/main.rs:100]: Looks like 2 lines above are not relevant anymore?

Comment (andybalaam) [reply src/main.rs:100]: Oops, yeah! Fix pushed in 7263b18 . Happy to rebase before merge if you'd like that.

**Comment by TatriX** [comment]
Thanks again! :)

**Comment by TatriX** [comment]
Oops, fixup didn't work as I expected. I've rebased manually and force pushed back to main.

**Comment by andybalaam** [comment]
Yeah, github doesn't squash fixups on merge, even though I think it definitely should...

**Pull request merged**